docs/releases/3.0.txt

```diff
@@ -106,7 +106,7 @@ Minor features
   <django.contrib.admin.ModelAdmin.get_inlines>` method allows specifying the
   inlines based on the request or model instance.
 
-* Select2 library is upgraded from version 4.0.3 to 4.0.7.
+* Select2 library is upgraded from version 4.0.3 to 4.0.12.
```

Comment:
Django 3.0 is in feature freeze. Can you revert this and add an entry to `docs/releases/3.1.txt`?